Curtains/Curtains.Infrastructure/Database/CurtainsDbContext.cs

```diff
@@ -3,18 +3,39 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Storage;
 
+using Curtains.Domain.Models;
+
 namespace Curtains.Infrastructure.Database
 {
     public class CurtainsDbContext : DbContext
     {
         private IDbContextTransaction _currentTransaction;
 
         #region DbSetsRegion
-
-        
-
+        public virtual DbSet<AccessoriesModel> Accessories { get; set; } = null!;
```

Comment:
Why you set null over all?

Reply:
As I understand ! is a null-forgiving operator and in this way we define that property can't be null

Reply:
it's not necessaty to use it in this way.